tests/foreman/api/test_convert2rhel.py

```diff
@@ -422,5 +428,10 @@ def test_convert2rhel_centos_with_pre_conversion_template_check(
     assert host_content['facts']['conversions::success'] == 'true'
     convert2rhel_facts = json.loads(centos.execute('cat /etc/rhsm/facts/convert2rhel.facts').stdout)
     assert convert2rhel_facts['conversions.env.CONVERT2RHEL_THROUGH_FOREMAN'] == '1'
-    assert target_os_name in convert2rhel_facts['conversions.target_os.name']
+    # https://issues.redhat.com/browse/RHELC-1737
+    assert (
+        "CentOS"
+        if is_open('RHELC-1737')
+        else target_os_name in convert2rhel_facts['conversions.target_os.name']
+    )
```

Comment:
It isn't good practice to modify assertion as per open issue, but when checking locally this assertion is always true, so I'd suggest to handle it in the variable instead of assertion, and I think it's best to check `'CentOS Linux'` instead of just CentOS for target_os.name 
```suggestion
    target_os_name = 'CentOS Linux' if is_open('RHELC-1737') else 'Red Hat'
    assert target_os_name in convert2rhel_facts['conversions.target_os.name']
```

Reply:
This is altered because of the open bug and I added a check for it. But you are right, we should not do incorrect assertions. I have skipped the assertion until bug is fixed.

Reply:
no, I don't mean to skip entire assertion until bug is fixed, instead we can check if we're getting same target OS as source OS until its fixed as suggested in the `target_os_name` variable and not in assertion

Reply:
Why should we assert the incorrect target_os_version? I think it is better to skip it. We do not expect os_version to be CentOS or Oracle post conversion. Wdyt?

Reply:
we've been validating this fact for a while for conversion tests, so I don't think it's good idea to skip it now, instead I'd suggest and think asserting both source_os and target_os is good addition to our conversion tests, and anyway It'll verify at least we're getting this facts after conversion

Reply:
Updated